README.md

```diff
@@ -6,35 +6,67 @@ A sidechain to Tezos by Marigold focused on higher throughput.
 
 ### Pre-requisites
 
-#### Esy Package Manager
+You'll need a one of the two package managers Deku is developed with:
 
-Marigold Sidechain is being developed with
-[esy](https://esy.sh/). `esy` can be installed with NPM or Yarn.
+#### With [Esy](https://esy.sh)
+Esy can be installed with NPM or Yarn.
 
 ```sh
 npm i -g esy # on Ubuntu you might need --unsafe-perms to work around EACCES issues
 # or
 yarn global add esy
 ```
 
-Alternatively, you can use the `nix` package manager with
-[flakes enabled](https://nixos.wiki/wiki/Flakes#Installing_flakes) to enter an
-environment preloaded with the correct dependencies, including `esy`:
+#### With Nix
 
+Nix can be installed with the following command on *nix systems:
```

Comment:
I think it is better to link the official installation guide and say something like
> Follow the [official guide](https://nixos.org/download.html) to install nix

Reply:
Smart thinking. Done.